### ISSUE TITLE
Architecture

### ISSUE THREAD

**Issue by SemenMartynov** [AndreiChugunov/jsonvalidator#7]
Andrei, I think you should think about your architecture when you have a time.
Make some wrapper-class over the server here looks like a good idea, don't you think?

https://github.com/AndreiChugunov/jsonvalidator/blob/170ce154fdc29ba39e0149fdd18e38e028952a1e/src/main/java/Main.java#L23

**Comment by AndreiChugunov** [comment]
@SemenMartynov  yeah! i ll fix this asap. Was thinking not about arcitacture at first but about the most simple solution.  Though, i want to redo the task on a fully functional language like Clojure, hope you ll check it out!

**Comment by SemenMartynov** [comment]
I can try, but I'm bad in Clojure.
But we can find someone who understands this better than me.
